packages/table/preview/index.tsx

```diff
@@ -246,6 +252,12 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
     }
 
     public componentWillUpdate(_nextProps: {}, nextState: IMutableTableState) {
+        // save off any changes to the state to local storage
```

Comment:
nit: I'd do this work in `componentDidUpdate`. Just makes more sense to me to change a thing and then have this code come along and say, "Oh, something change**d**? Let's save it."